templates/base.html

```diff
@@ -8,7 +8,12 @@
     <body>
         <div>
             <a href="/"> Homepage </a>
-            <a href="contact/"> Contact </a>
+            <a href="/contact"> Contact </a>
```

Comment:
Please define route namespaces and names instead of using directly the route path.